gno.land/cmd/gnoland/testdata/realm_banker_issued_coin_denom.txtar

```diff
@@ -61,12 +80,12 @@ import (
 
 func Mint(addr std.Address, denom string, amount int64) {
 	banker := std.GetBanker(std.BankerTypeRealmIssue)
-	banker.IssueCoin(addr, denom, amount)
+	banker.IssueCoin(addr, std.CurrentRealm().Denom(denom), amount)
```

Comment:
`Denom(denom)` might be a bit confusing; also, not sure if a realm should have a denom. I think we might need a better name for this function.

Also, if you're adding a new function to `std`, please update the reference docs (docs/reference/...)

Reply:
I have few proposals:

```go
std.CurrentRealm().ComposeDenom(denom string) // i like this one
std.CurrentRealm().ResolveDenom(denom string)
std.CurrentRealm().DeriveDenom(denom string)
```

What do you think about this ?
I will update reference when i get the right name to fix everything in one-time, thanks for review :heart: